1.3/Defs/Morphs/Animal/Avian/Avian_Parts.xml

```diff
@@ -1,22 +1,99 @@
 <Defs xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../Schemas/MutationDef.xsd">
 	<Pawnmorph.Hediffs.MutationDef Name ="AvianJawPart" ParentName="AnimalJawPart" Abstract="true">
-		<description>A morph beak.</description>
 		<mutationMemory>EtherBeak</mutationMemory>
+		<comps>
+			<li Class="HediffCompProperties_VerbGiver">
```

Comment:
Does every avian have a beak attack? Though I guess we can always adjust later if needed

Reply:
We'll need to figure this out in testing. I don't believe he intend to give every avian a combat bonus, and usually combat verbs are indicated with a dps increase stat.